websites-css/aboutus.css

```diff
@@ -1,3 +1,20 @@
+:root {
+    --color-bg: #223;
+    --color-bg-text: #ffffff;
+    --color-header-bg: #223;
+    --color-header-text: #ffffff;
+    --color-button: #E1A95F;
+    --color-button-hover: #C89045;
+    --color-button-text: #2C2C2C;
+    --color-searchbar: #151520;
+    --color-content-section: #3a3a4d;
+    --font-size-header: 1.2rem;
+    --font-size-h2: 2.5rem;
+    --font-size-h3: 1.8rem;
+    --font-size-p: 1.2rem;
+    --font-family: "Roboto", sans-serif;
+  }
```

Comment:
you've included :root in your CSS styles. However, it's not necessary to include it since it is already defined in our master CSS file.

Reply:
forgot to delete, thanks!!